app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -144,30 +144,6 @@ public void validate(Object obj, Errors errors) {
         validateEmails(errors, study.getConsentNotificationEmail(), "consentNotificationEmail");
         validateDataGroupNamesAndFitForSynapseExport(errors, study.getDataGroups());
         
-        // emailVerificationEnabled=true (public study):
-        //     externalIdValidationEnabled and externalIdRequiredOnSignup can vary independently
-        // emailVerificationEnabled=false:
-        //     externalIdValidationEnabled and externalIdRequiredOnSignup must both be true
-        if (!study.isEmailVerificationEnabled()) {
-            if (!study.isExternalIdRequiredOnSignup()) {
-                errors.rejectValue("externalIdRequiredOnSignup", "cannot be disabled if email verification has been disabled");
-            }
-            if (!study.isExternalIdValidationEnabled()) {
-                errors.rejectValue("externalIdValidationEnabled", "cannot be disabled if email verification has been disabled");
-            }
-        }
-        
-        // Links in installedLinks are length-constrained by SMS.
-        if (!study.getInstallLinks().isEmpty()) {
-            for (Map.Entry<String,String> entry : study.getInstallLinks().entrySet()) {
-                if (StringUtils.isBlank(entry.getValue())) {
-                    errors.rejectValue("installLinks", "cannot be blank");
-                } else if (entry.getValue().length() > BridgeConstants.SMS_CHARACTER_LIMIT) {
-                    errors.rejectValue("installLinks", "cannot be longer than "+BridgeConstants.SMS_CHARACTER_LIMIT+" characters");
-                }
-            }
-        }
```

Comment:
It looks like we deleted the install links from the validator. Given that we still have unit tests for these, this seems unintentional.